src/extensions/assembly-symmetry/prop.ts

```diff
@@ -74,11 +74,8 @@ export namespace AssemblySymmetryData {
     export const DefaultServerUrl = 'https://data.rcsb.org/graphql'; // Alternative: 'https://www.ebi.ac.uk/pdbe/aggregated-api/pdb/symmetry' (if serverType is 'pdbe')
 
     export function isApplicable(structure?: Structure): boolean {
-        return (
-            !!structure && structure.models.length === 1 &&
-            Model.hasPdbId(structure.models[0]) &&
-            isBiologicalAssembly(structure)
-        );
+        if (!structure || !Model.hasPdbId(structure.models[0])) return false;
```

Comment:
Looks like you're accessing `structure.models[0]` before you checked `structure.models.length === 1`. That seems wrong.

Reply:
Should there be a check to ensure that at least 1 model exists in the array? `if (!structure || !structure.models.length || !Model.hasPdbId(structure.models[0])) return false;`

Reply:
You are checking if `structure.models.length===1` on the next line.

So maybe it could reorganize the condition somehow.

I'm not sure though what should happen if `structure.models.length>1 && Model.isIntegrative(structure.models[0])` (or if it even possible)

Reply:
You're correct. I was under the impression that `models` would have multiple elements for NMR and multi-model IHMs. [c1e33fa](https://github.com/molstar/molstar/pull/1506/commits/c1e33fac94bc5a0f5254c9a08052d03ccb6762d5) brings back the `structure.models.length === 1` check in its original position. Nice catch.